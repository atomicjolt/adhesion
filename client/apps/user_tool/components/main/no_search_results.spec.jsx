import React from 'react';
import { shallow } from 'enzyme';

import NoSearchResults from './no_search_results';

describe('NoSearchResults', () => {
  it('renders the no search results section', () => {
    const result = shallow(<NoSearchResults searchTerm="some search" />);

    expect(result).toMatchSnapshot();
  });
});
