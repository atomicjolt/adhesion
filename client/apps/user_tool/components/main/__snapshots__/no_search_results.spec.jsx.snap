// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoSearchResults renders the no search results section 1`] = `
<section>
  <p>
    Your search - 
    <strong>
      some search
    </strong>
     - did not match any users.
  </p>
  <p>
    Suggestions:
  </p>
  <ul>
    <li>
      Double check your spelling.
    </li>
    <li>
      Try more general keywords.
    </li>
    <li>
      Try different keywords.
    </li>
  </ul>
</section>
`;
